src/components/Editor/Tabs.css

```diff
@@ -67,13 +67,13 @@
   fill: var(--theme-body-color);
 }
 
-.source-tab .prettyPrint {
-  line-height: 0;
-}
-
-.source-tab .prettyPrint svg {
+.source-tab img.prettyPrint {
+  mask: url("/images/prettyPrint.svg") no-repeat;
```

Comment:
lets always use a `/` i.e. `mask: url("/images/prettyPrint.svg") no-repeat;` 

so lets change the other sites